src/HumanView.cpp

```diff
@@ -96,15 +282,72 @@ void HumanView::drawObjects() {
     display.display();
 }
 
-void HumanView::checkKeyboardStart() {
+void HumanView::checkKeyboardStart(float time) {
     if (sf::Keyboard::isKeyPressed(sf::Keyboard::Enter)) {
-        //set gameState to running
-        logic.setGameState(1);   
+        //set gameState to levelDialogue
+        logic.setGameState(4);
+        startTime = time;
+        currTime = time;
+        first = true;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::S)) {
+        //set gameState to settings
+        logic.setGameState(3);
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num0)) {
+        //change currentLevel to 0
+        currentLevel = 0;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num1)) {
+        //change currentLevel to 1
+        currentLevel = 1;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num2)) {
+        //change currentLevel to 2
+        currentLevel = 2;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num3)) {
+        //change currentLevel to 3
+        currentLevel = 3;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num4)) {
+        //change currentLevel to 4
+        currentLevel = 4;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num5)) {
+        //change currentLevel to 5
+        currentLevel = 5;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num6)) {
+        //change currentLevel to 6
+        currentLevel = 6;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num7)) {
+        //change currentLevel to 1
+        currentLevel = 7;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num8)) {
+        //change currentLevel to 8
+        currentLevel = 8;
+    }
+    if (sf::Keyboard::isKeyPressed(sf::Keyboard::Num9)) {
+        //change currentLevel to 9
+        currentLevel = 9;
     }
 }
 
-void HumanView::checkKeyboardEndLevel() {
-
+void HumanView::checkKeyboardEndLevel(float time) {
```

Comment:
Add option to go back to menu screen or continue to next level when the player reaches the score screen after each level